controllers/token.go

```diff
@@ -30,8 +30,9 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
-const (
-	defaultTokenTTL = 10 * time.Minute
+var (
+	// DefaultTokenTTL is the amount of time a bootstrap token (and therefore a KubeadmConfig) will be valid
+	DefaultTokenTTL = 10 * time.Minute
```

Comment:
I think bumping this to 15m and leaving the sync interval at 10m is probably fine

Reply:
Yes, 9 minutes is too uneven for me as well 😂